src/main/java/com/coveros/selenified/element/check/Matches.java

```diff
@@ -105,25 +105,25 @@ default String checkText(String expectedPattern, double waitFor, double timeTook
      * @param timeTook - the amount of time it took for wait for something (assuming we had to wait)
      * @return String: the actual text of the table cell. null will be returned if the element isn't present or a table, or table cell doesn't exist
      */
-    default String checkText(int row, int col, String pattern, double waitFor, double timeTook) {
+    String checkText(int row, int col, String pattern, double waitFor, double timeTook) {
         String column = " and column ";
         String within = " within element ";
         // record the action
-        if (!isPresentTable("Expected to find cell at row " + row + column + col + within + getElement().prettyOutput() +
+        if (!isPresentTable("Expected to find cell at row " + row + column + col + within + this.element.prettyOutput() +
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "Expected to find cell at row " 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/sonar/coding_rules#rule_key=squid%3AS1192)